.dockstore.yml

```
@@ -0,0 +1,6 @@
+# Dockstore config version, not pipeline version
+version: 1.2
+workflows:
+  - subclass: nfl
+    primaryDescriptorPath: /nextflow.config
+    publish: True
```

We should add the author list here. It will be displayed on dockstore website. As per doc

> # An optional array of authorship information.
    # Note that if orcid is present, then all other fields will be ignored, as information will be taken from orcid.
    # If orcid is not present, make sure to at a minimum include the name field for each author.
    authors:
      - name: <String>
        orcid: <String>
        email: <String>
        role: <String>
        affiliation: <String>

```suggestion
    publish: True
    authors:
      - name: Justin Jeyakani
        orcid: 
      - name: Mar Gonzalez-Porta
        orcid: 
      - name: Maxime HEBRARD
        orcid: 0000-0003-2346-4513
      - name: Rodrigo Toro
        orcid:
      - name: Nicolas Bertin
        orcid:
```